experimental/tsconfig.json

```diff
@@ -3,13 +3,13 @@
   "files": [],
   "references": [
     {
-      "path": "packages/opentelemetry-api-metrics/tsconfig.all.json"
```

Comment:
Submitted https://github.com/open-telemetry/opentelemetry-js/pull/2765 to fix these autofixes.